test/data/atom10_2.xml

```diff
@@ -55,4 +55,13 @@
     <media:player url="http://example.com/player.html"/>
   </entry>
 
+  <entry>
```

Comment:
Shouldn't we test this for RSS too?

Reply:
Makes sense, I've added a similar test case for one of hte rss variants